packages/components/src/tooltip/test/index.js

```diff
@@ -11,60 +11,55 @@ import Tooltip from '../';
  * WordPress dependencies
  */
 import { TOOLTIP_DELAY } from '../index.js';
-import { act } from '@testing-library/react';
 
 describe( 'Tooltip', () => {
 	describe( '#render()', () => {
 		it( 'should render children (abort) if multiple children passed', () => {
-			// Mount: Enzyme shallow does not support wrapping multiple nodes.
-			const wrapper = mount(
+			const { container } = render(
 				<Tooltip>
 					<div />
 					<div />
 				</Tooltip>
 			);
 
-			expect( wrapper.children() ).toHaveLength( 2 );
+			expect( container.children ).toHaveLength( 2 );
```

Comment:
AFAIK DOM traversing is not recommended - this one should be easy to fix if we replace the `div`s with some elements that we can query by `role` for example. Just like we did for the button test below.

Reply:
Also I believe the point of this specific test is that the tooltip won't be rendered if [multiple children are passed](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/tooltip/README.md#children). In other words, we want to know explicitly that the **tooltip is not rendered**, not just that the children are rendered as is.

So the test snippet could look something like this for example:

```jsx
		<Tooltip text="foo">
			<button>Button 1</button>
			<button>Button 2</button>
		</Tooltip>
```

focus the first button, and assert that the "foo" text is not in the DOM.

Reply:
Fixed by 28b1abe 👍